<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGraspEditor</class>
 <widget class="QMainWindow" name="MainWindowGraspEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> VirtualRobot - GraspEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadObject">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Select ManipulationObject File</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxEEF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>End Effector</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Grasp</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxGrasp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAddGrasp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>460</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Grasp</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadRobot">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Robot File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Close EEF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Open EEF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRenameGrasp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>420</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rename Grasp</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderX">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>530</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Move Grasp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>530</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderY">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>550</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>550</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderZ">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>570</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>570</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderRo">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>590</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>590</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ro</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderPi">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>610</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>610</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pi</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSliderYa">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>630</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>630</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ya</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>660</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Visualization</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxColModel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>680</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Collision Model</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxGraspSet">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>700</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Grasp Set</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxTCP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>720</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show TCP Coord System</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>pushButtonLoadObject</tabstop>
  <tabstop>pushButtonLoadRobot</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>comboBoxEEF</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>pushButtonOpen</tabstop>
  <tabstop>comboBoxGrasp</tabstop>
  <tabstop>pushButtonRenameGrasp</tabstop>
  <tabstop>pushButtonAddGrasp</tabstop>
  <tabstop>horizontalSliderX</tabstop>
  <tabstop>horizontalSliderY</tabstop>
  <tabstop>horizontalSliderZ</tabstop>
  <tabstop>horizontalSliderRo</tabstop>
  <tabstop>horizontalSliderPi</tabstop>
  <tabstop>horizontalSliderYa</tabstop>
  <tabstop>checkBoxColModel</tabstop>
  <tabstop>checkBoxGraspSet</tabstop>
  <tabstop>checkBoxTCP</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
